src/Ombi/ClientApp/app/search/music/musicsearch.component.html

```diff
@@ -1,7 +1,7 @@
 ﻿<!-- Movie tab -->
 <div role="tabpanel" class="tab-pane active" id="MoviesTab">
     <div class="input-group">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
+        <input id="search" type="text" placeholder="Type Here to Search" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
```

Comment:
Same here regarding the placeholder translation

Reply:
Added